docs/terraform.md

```diff
@@ -1,104 +1 @@
-| use_existing_security_group | If set to `true`, will use variable `workers_security_group_id` to run EKS workers using an existing security group that was created outside of this module, workaround for errors like `count cannot be computed` | bool | `false` | no |
-| vpc_id | VPC ID for the EKS cluster | string | - | yes |
-| wait_for_capacity_timeout | A maximum duration that Terraform should wait for ASG instances to be healthy before timing out. Setting this to '0' causes Terraform to skip all Capacity Waiting behavior | string | `10m` | no |
-| wait_for_elb_capacity | Setting this will cause Terraform to wait for exactly this number of healthy instances in all attached load balancers on both create and update operations. Takes precedence over `min_elb_capacity` behavior | number | `0` | no |
-| workers_role_policy_arns | List of policy ARNs that will be attached to the workers default role on creation | list(string) | `<list>` | no |
-| workers_role_policy_arns_count | Count of policy ARNs that will be attached to the workers default role on creation. Needed to prevent Terraform error `count can't be computed` | number | `0` | no |
-| workers_security_group_id | The name of the existing security group that will be used in autoscaling group for EKS workers. If empty, a new security group will be created | string | `` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| autoscaling_group_arn | ARN of the AutoScaling Group |
-| autoscaling_group_default_cooldown | Time between a scaling activity and the succeeding scaling activity |
-| autoscaling_group_desired_capacity | The number of Amazon EC2 instances that should be running in the group |
-| autoscaling_group_health_check_grace_period | Time after instance comes into service before checking health |
-| autoscaling_group_health_check_type | `EC2` or `ELB`. Controls how health checking is done |
-| autoscaling_group_id | The AutoScaling Group ID |
-| autoscaling_group_max_size | The maximum size of the AutoScaling Group |
-| autoscaling_group_min_size | The minimum size of the AutoScaling Group |
-| autoscaling_group_name | The AutoScaling Group name |
-| launch_template_arn | ARN of the launch template |
-| launch_template_id | The ID of the launch template |
-| security_group_arn | ARN of the worker nodes Security Group |
-| security_group_id | ID of the worker nodes Security Group |
-| security_group_name | Name of the worker nodes Security Group |
-| workers_role_arn | ARN of the worker nodes IAM role |
-| workers_role_name | Name of the worker nodes IAM role |
-
+Error: template: inputs:10:118: executing "inputs" at <.GetValue>: error calling GetValue: json: error calling MarshalJSON for type types.String: invalid character '.' in string escape code
```

Comment:
@tthayer please have a look at the inputs. Something is malformed causing this error.

Reply:
(it's also possible that it's an existing input you didn't change - we upgraded `terraform-docs` which might lead to that)